src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx

```diff
@@ -13,6 +15,18 @@ export interface ISocialLinkData {
   label: string;
 }
 
+const SFlexRow = styled(FlexRow)`
+  @media (max-width: 768px) {
```

Comment:
you can use the breakpoint value from the `constants/grid` file

Reply:
👍